hera_corr_cm/hera_corr_cm.py

```diff
@@ -249,7 +281,7 @@ def stop_taking_data(self):
             return ERROR
```

Comment:
I know doing `return ERROR` and `return OK` is a hold-over from Jack, but can we change this to use `return False` and `return True` instead?

Reply:
I'm fine with this change. I think we import corr_cm.ERROR somewhere in M&C so we'll need to fix that too